aws/request_test.go

```diff
@@ -207,7 +213,9 @@ func TestRequestExhaustRetries(t *testing.T) {
 		{StatusCode: 500, Body: body(`{"__type":"UnknownError","message":"An error occurred."}`)},
 	}
 
-	s := awstesting.NewClient(unit.Config())
+	cfg := unit.Config()
+	cfg.Retryer = aws.NewDefaultRetryer()
+	s := awstesting.NewClient(cfg)
```

Comment:
Did `unit.Config` not already include a retryer? Its not clear from this why it needed to be added. I would of expected the retryer to of been either already set in `unit.Config` or when the below `awstesting.NewClient` is created.

Reply:
unit.Config didn't include the retryer previously. I added a retryer in unit.config with this PR, but didn't remove the redundant changes which I made earlier.